aries-site/src/pages/templates/forms.mdx

```diff
@@ -213,11 +198,40 @@ For standard forms, use an asterisk(\*) to indicate required field/controls.
   <SingleFormFieldExample />
 </Example>
 
-## Customize
+### Customize
 
 <Example
   code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/forms/CustomizeExample.js"
   figma="https://www.figma.com/file/3fkwBelW5UsCbfwdDCJkT8/HPE-Form-Templates?node-id=54%3A608"
 >
   <CustomizeExample />
 </Example>
+
+### Accessibility
+
+All form input elements must be contained within a FormField.
+
+**FormFields must include:** 
+
+- A "for" property that matches the id of its child input. If using Grommet, this is applied with the htmlFor prop.
+- A `name` property applied on both the FormField and its child input element.
+- A clear, concise label.
+
+## Exceptions and how to address them
+
+- A login/registration form, which typically shows two to three required fields do not need to be annotated in any way - the form is short and the necessity of completing the fields is standard practice.
+
+   - For login/registration forms, apply `required={{ indicator: false }}` to the FormFields to remove the required indicator.
+
+- Forms with minimal fields are optimal but occasionally, capturing extra information would be highly beneficial for the user. The following are some options for addressing these cases: 
+
+   - Create the form the optimal way; with only the minimum required fields so that the user can focus on completing their task. Separately, in a preferences or settings dashboard, for example, create those fields where the user can proactively add this helpful information at another time. An example of a helpful but not required field could be asking for someone's official title or role. It might be highly useful to capture it right away on a sign-in form, but a title or role might be too problematic for the admin to enter. Instead, offer this "role" or "title" field in profile settings.
```

Comment:
```suggestion
   - Create the form the optimal way; with only the minimum required fields so that the user can focus on completing their task. Separately, in a preferences or settings dashboard, for example, provide a form that allows the user to add helpful, optional information at another time. An example of a helpful but not required field could be asking for someone's official title or role. If it is not necessary to capture this information right away on a sign-in form, offer this "role" or "title" field in profile settings.
```